src/System.Reflection.TypeExtensions/pkg/System.Reflection.TypeExtensions.pkgproj

```diff
@@ -27,4 +26,18 @@
     <SuppressMetaPackage Include="NETStandard.Library" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  
+  <Target Name="_WorkaroundWrongNCA10Asset" AfterTargets="HarvestStablePackage">
+    <HarvestPackage PackageId="System.Reflection.TypeExtensions" 
+                    PackageVersion="4.4.0" 
```

Comment:
Note that once we update 'external\harvestPackages\harvestPackages.props' we'll need to also include 4.4.0 (under a condition).